google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/descriptor/AppEngineWebSchemaProvider.java

```diff
@@ -73,10 +74,14 @@ public XmlFile getSchema(@NotNull @NonNls String url, @Nullable Module module,
       if (facet != null) {
         final File file;
         if (isApplicationXmlFile(baseFile)) {
-          file = facet.getSdk().getApplicationSchemeFile();
+          file = CloudSdkService.getInstance().getApplicationSchemeFile();
         } else {
-          file = facet.getSdk().getWebSchemeFile();
+          file = CloudSdkService.getInstance().getWebSchemeFile();
```

Comment:
I was wondering that too. Maybe appengine-application.xml is an old concept? This is what the template looks like:

```
<appengine-application xmlns="http://appengine.google.com/ns/1.0">
  <application>app</application>
</appengine-application>
```